docs/LayoutWithFlexbox.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: basics-layout
+id: layout-with-flexbox
 title: Layout with Flexbox
 layout: docs
 category: The Basics
 permalink: docs/flexbox.html
```

Comment:
You may need to update the permalink to docs/layout-with-flexbox.html as well, as loading docs/flexbox.html throws a 500:

```
500 Error: Error: Following module not in specified search paths: /Users/hramos/git/react/react-native/website/src/react-native/docs/layout-with-flexbox.js
```